com.unity.render-pipelines.high-definition/Runtime/Lighting/LightLoop/LightLoop.cs

```diff
@@ -203,20 +204,30 @@ public enum TileClusterCategoryDebug : int
         /// <summary>Area lights.</summary>
         Area = 2,
         /// <summary>Area and punctual lights.</summary>
+        [InspectorName("Area and Punctual")]
         AreaAndPunctual = 3,
         /// <summary>Environment lights.</summary>
+        [InspectorName("Reflection Probes")]
         Environment = 4,
         /// <summary>Environment and punctual lights.</summary>
+        [InspectorName("Reflection Probes and Punctual")]
```

Comment:
@JulienIgnace-Unity , here is were I create the string of the enum with the correct naming